src/base/bittorrent/session.cpp

```diff
@@ -417,6 +417,13 @@ Session::Session(QObject *parent)
     , m_utpMixedMode(BITTORRENT_SESSION_KEY("uTPMixedMode"), MixedModeAlgorithm::TCP
         , clampValue(MixedModeAlgorithm::TCP, MixedModeAlgorithm::Proportional))
     , m_multiConnectionsPerIpEnabled(BITTORRENT_SESSION_KEY("MultiConnectionsPerIp"), false)
+    , m_validateHTTPSTrackerCertificate(BITTORRENT_SESSION_KEY("ValidateHTTPSTrackerCertificate"), false)
+    , m_httpsTrackerTLSVersion(BITTORRENT_SESSION_KEY("HTTPSTrackerTLSVersion"), TLSVersion::TLS12
```

Comment:
You forgot to change this and the following lines accordingly to changing enum type.

Reply:
I've dropped the TLSVersion:: now that they are unscoped but am still getting a plethora of build errors. What else needs to be changed?

Reply:
In order to store enum in CachedSettingsValue you need to store it either as int or as QString (manually converting) or as QMetaEnum (declaring it with Q_ENUM).